Data/core_rulebook/cr_abilities.lua

```diff
@@ -2832,10 +2832,11 @@ DefineAbility({
   Category="Intelligent Item",
   AllowMultiple=true,
   Stackable=false,
-  Chooses={
-    ChooseSpell(function (character, spell)
-      return (true and spell.Level <= 0 and spell.Level >= 0)
+  Choice={
+    Choose=ChooseSpell(function (character, spell)
+      return ((true and (((spell.Level <= 0) and (spell.Level >= 0)))))
```

Comment:
What is the "true an" coming from? Is that "ANY"?

Reply:
yea, "true" == ANY or ALL

Reply:
this looks like an `ANY[LEVELMAX=0;LEVELMIN=0]`